crate/cli/src/actions/cover_crypt/policy.rs

```diff
@@ -93,19 +91,39 @@ impl TryFrom<&Policy> for PolicySpecifications {
 
     fn try_from(policy: &Policy) -> Result<Self, Self::Error> {
```

Comment:
It seems you need to clone the `Policy` for this conversion (you are cloning many things out of it). If that is the case, the best would be to make it clear by taking ownership of it.